src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(operation, number) {
+      this.result = operation(this.result, number);
+
+      return this;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(initial, toOperate) {
+      return initial + toOperate;
+    },
```

Comment:
each function just accept 2 arguments, maybe it's not so clear why one of them is initial, and one of them is toOperate
what does it mean? let's simplify variable names, it's just 2 arguments, some 2 values that we need to proceed